src/libs/Navigation/shouldSetScreenBlurred/index.tsx

```diff
@@ -0,0 +1,8 @@
+/**
+ * @param navigationIndex
+ *
+ * Decides whether to set screen to blurred state
+ */
```

Comment:
Can you please add a comment here and to the native file why there is the difference between the comparison? why is one >1 and other =>1? Thanks!